pymc/distributions/truncated.py

```diff
@@ -195,11 +212,12 @@ def rv_op(cls, dist, lower, upper, max_n_steps, size=None):
                 cdf_lower_,
                 cdf_upper_,
                 rng=rng,
-                size=rv_inputs_[0],
+                size=rv_.shape,
             )
-            truncated_rv_ = icdf(rv_, uniform_)
+            truncated_rv_ = icdf(rv_, uniform_, warn_rvs=False)
```

Comment:
What does the `warn_rvs=False` do here?

Reply:
It emits a warning if there are random variables in the icdf graph. In this case there is one and it is supposed to be there (the uniform) since the icdf is part of the forward graph itself.